templates/dma.conf.j2

```diff
@@ -11,7 +13,7 @@ AUTHPATH /etc/dma/auth.conf
 {% if dma_starttls %}
 STARTTLS
 {% endif %}
-MAILNAME {{ ansible_nodename }}
+MAILNAME {{ dma_mailname |default(ansible_nodename) }}
```

Comment:
Would you mind adding an additional space between `|` and `default`? In the commit message you said this would need further testing. Have you been able to test it successfully? From what I can see in the code, this should work.

Reply:
Hi @niklaas ,
I've added the space in my latest commit. I haven't yet had a chance to test the role again - i want to run a disposable server to test it in - so I have only tested the case where dma_mailname exists.

Reply:
@niklaas 
I just did another run and when dma_mailname is unset the default to ansible_nodename works correctly.

Karl.